30_data_summarize/src/site_summary.R

```diff
@@ -0,0 +1,33 @@
+#one row per site, including ID, lat/lon, site type, maybe n records from the data pull
+
+summarize_site_observations <- function(target_ind, daily_flow_ind, 
+                                        dv_inventory, uv_inventory) {
+  
+  daily_flow <- readRDS(sc_retrieve(daily_flow_ind, remake_file = '20_data_munge.yml')) 
+  #daily_flow <- readRDS('20_data_munge/out/daily_flow.rds')
+  #dv_inventory <- readRDS('10_nwis_pull/inout/nwis_dv_inventory.rds')
+  #uv_inventory <- readRDS('10_nwis_pull/inout/nwis_uv_inventory.rds')
+  #dv_inventory <- readRDS(sc_retrieve(dv_inventory_ind, remake_file = '20_data_munge.yml'))
+  #uv_inventory <- readRDS(sc_retrieve(uv_inventory_ind, remake_file = '20_data_munge.yml'))
+  
+  
+  daily_summary <- daily_flow %>% 
+    group_by(site_id, source) %>% 
+    summarize(n_days = n(),
+              median_flow = median(flow_cfs))
+  stopifnot(length(unique(daily_summary$site_id)) == nrow(daily_summary))
```

Comment:
This assumes there is only one "source" per site. Just want to check - that is what we want, correct?

Reply:
I think so, `daily_flow` is the aggregated uv + dv file, so I'm assuming redundant sources have already been dealt with.

Reply:
That sounds right. If not, then we have a site that has data from both uv and dv which is not how this is supposed to work, so would be good to stop the code and troubleshoot.